_data/tools.yml

```
@@ -2639,3 +2639,14 @@
   v2: true
   v3: true
 
+- name: API Insights
+  category:
+    - monitoring
+    - description-validators
+  link: https://apiinsights.io/
+  github: https://github.com/treblle
+  description:
+    API Insights is a web application and native Mac application that allows you get real time insights into your API using your OpenAPI specification. Think of it like a static analysis tool for your API, using your OpenAPI Specification file.
+  v2: false
+  v3: true
+
```

Do you want to mark v3.1 support too? 
```suggestion
v3_1: true

```